unlock-app/src/components/content/certification/CertificationDetails.tsx

```diff
@@ -332,6 +332,8 @@ export const CertificationDetails = ({
     ? key?.expiration
     : expirationAsDate(key?.expiration)
 
+  const isMobile = window?.innerWidth < 768
```

Comment:
ok can you make sure this also looks good on width larger than that but smaller than full screen?

Reply:
Also, can you mive this logic _inside_ the certificate component?

Reply:
I think it will break in case we try to send certification via mail because `window` will not be defined there

Reply:
ha good point! ok ok , please send screenshots for intermediate sizes?

Reply:
<img width="600" alt="Screenshot 2023-04-17 at 21 11 10" src="https://user-images.githubusercontent.com/20865711/232587408-e3d5a362-a23d-4357-b27e-2c7d303bfa8c.png">

<img width="600" alt="Screenshot 2023-04-17 at 21 11 06" src="https://user-images.githubusercontent.com/20865711/232587420-6c45bb4b-3c2d-46b2-b44c-ec2e68b4178e.png">

<img width="600" alt="Screenshot 2023-04-17 at 21 11 00" src="https://user-images.githubusercontent.com/20865711/232587428-90ce93fe-2477-4c61-b6ea-867c75c2e0de.png">